### PR TITLE
infoschema: add column comments for statement summary tables

### PR DESCRIPTION
**Pull request by djshow832**
### What problem does this PR solve?

Problem Summary:
It is suggested that add the definition of each column to the table definition, rather than in the document. Because the document is too long.

### What is changed and how it works?

What's Changed:
Define comments for each column.

How it Works:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Manual test (add detailed scripts or steps below)
```
mysql> show create table performance_schema.events_statements_summary_by_digest;
| Table                               | Create Table
| events_statements_summary_by_digest | CREATE TABLE `events_statements_summary_by_digest` (
  `SUMMARY_BEGIN_TIME` timestamp(6) NOT NULL COMMENT 'Begin time of this summary',
  `SUMMARY_END_TIME` timestamp(6) NOT NULL COMMENT 'End time of this summary',
  `STMT_TYPE` varchar(64) NOT NULL COMMENT 'Statement type',
  `SCHEMA_NAME` varchar(64) DEFAULT NULL COMMENT 'Current schema',
  `DIGEST` varchar(64) NOT NULL,
  `DIGEST_TEXT` longtext NOT NULL COMMENT 'Normalized statement',
  `TABLE_NAMES` text DEFAULT NULL COMMENT 'Involved tables',
  `INDEX_NAMES` text DEFAULT NULL COMMENT 'Used indices',
  `SAMPLE_USER` varchar(64) DEFAULT NULL COMMENT 'Sampled user who executed this statement',
  `EXEC_COUNT` bigint(20) unsigned NOT NULL COMMENT 'Count of execution',
  `SUM_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Sum latency of this statement',
  `MAX_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Max latency of this statement',
  `MIN_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Min latency of this statement',
  `AVG_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Average latency of this statement',
  `AVG_PARSE_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Average latency of parsing',
  `MAX_PARSE_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Max latency of parsing',
  `AVG_COMPILE_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Average latency of compiling',
  `MAX_COMPILE_LATENCY` bigint(20) unsigned NOT NULL COMMENT 'Max latency of compiling',
  `COP_TASK_NUM` bigint(20) unsigned NOT NULL COMMENT 'Total number of CopTasks',
  `AVG_COP_PROCESS_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average processing time of CopTasks',
  `MAX_COP_PROCESS_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max processing time of CopTasks',
  `MAX_COP_PROCESS_ADDRESS` varchar(256) DEFAULT NULL COMMENT 'Address of the CopTask with max processing time',
  `AVG_COP_WAIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average waiting time of CopTasks',
  `MAX_COP_WAIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max waiting time of CopTasks',
  `MAX_COP_WAIT_ADDRESS` varchar(256) DEFAULT NULL COMMENT 'Address of the CopTask with max waiting time',
  `AVG_PROCESS_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average processing time in TiKV',
  `MAX_PROCESS_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max processing time in TiKV',
  `AVG_WAIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average waiting time in TiKV',
  `MAX_WAIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max waiting time in TiKV',
  `AVG_BACKOFF_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average waiting time before retry',
  `MAX_BACKOFF_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max waiting time before retry',
  `AVG_TOTAL_KEYS` bigint(20) unsigned NOT NULL COMMENT 'Average number of scanned keys',
  `MAX_TOTAL_KEYS` bigint(20) unsigned NOT NULL COMMENT 'Max number of scanned keys',
  `AVG_PROCESSED_KEYS` bigint(20) unsigned NOT NULL COMMENT 'Average number of processed keys',
  `MAX_PROCESSED_KEYS` bigint(20) unsigned NOT NULL COMMENT 'Max number of processed keys',
  `AVG_PREWRITE_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average time of prewrite phase',
  `MAX_PREWRITE_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max time of prewrite phase',
  `AVG_COMMIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average time of commit phase',
  `MAX_COMMIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max time of commit phase',
  `AVG_GET_COMMIT_TS_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average time of getting commit_ts',
  `MAX_GET_COMMIT_TS_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max time of getting commit_ts',
  `AVG_COMMIT_BACKOFF_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average time before retry during commit phase',
  `MAX_COMMIT_BACKOFF_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max time before retry during commit phase',
  `AVG_RESOLVE_LOCK_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average time for resolving locks',
  `MAX_RESOLVE_LOCK_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max time for resolving locks',
  `AVG_LOCAL_LATCH_WAIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Average waiting time of local transaction',
  `MAX_LOCAL_LATCH_WAIT_TIME` bigint(20) unsigned NOT NULL COMMENT 'Max waiting time of local transaction',
  `AVG_WRITE_KEYS` double unsigned NOT NULL COMMENT 'Average count of written keys',
  `MAX_WRITE_KEYS` bigint(20) unsigned NOT NULL COMMENT 'Max count of written keys',
  `AVG_WRITE_SIZE` double NOT NULL COMMENT 'Average amount of written bytes',
  `MAX_WRITE_SIZE` bigint(20) unsigned NOT NULL COMMENT 'Max amount of written bytes',
  `AVG_PREWRITE_REGIONS` double NOT NULL COMMENT 'Average number of involved regions in prewrite phase',
  `MAX_PREWRITE_REGIONS` int(11) unsigned NOT NULL COMMENT 'Max number of involved regions in prewrite phase',
  `AVG_TXN_RETRY` double NOT NULL COMMENT 'Average number of transaction retries',
  `MAX_TXN_RETRY` int(11) unsigned NOT NULL COMMENT 'Max number of transaction retries',
  `SUM_BACKOFF_TIMES` bigint(20) unsigned NOT NULL COMMENT 'Sum of retries',
  `BACKOFF_TYPES` varchar(1024) DEFAULT NULL COMMENT 'Types of errors and the number of retries for each type',
  `AVG_MEM` bigint(20) unsigned NOT NULL COMMENT 'Average memory(byte) used',
  `MAX_MEM` bigint(20) unsigned NOT NULL COMMENT 'Max memory(byte) used',
  `AVG_AFFECTED_ROWS` double unsigned NOT NULL COMMENT 'Average number of rows affected',
  `FIRST_SEEN` timestamp(6) NOT NULL COMMENT 'The time these statements are seen for the first time',
  `LAST_SEEN` timestamp(6) NOT NULL COMMENT 'The time these statements are seen for the last time',
  `QUERY_SAMPLE_TEXT` longtext DEFAULT NULL COMMENT 'Sampled original statement',
  `PREV_SAMPLE_TEXT` longtext DEFAULT NULL COMMENT 'The previous statement before commit',
  `PLAN_DIGEST` varchar(64) DEFAULT NULL COMMENT 'Digest of its execution plan',
  `PLAN` longtext DEFAULT NULL COMMENT 'Sampled execution plan'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
```

Side effects

N/A

### Release note 

N/A

### CODE REVIEW

File: infoschema/perfschema/const.go [hunk 1]
```diff
@@ -385,72 +385,72 @@ const tableStagesHistoryLong = "CREATE TABLE if not exists performance_schema."
 	"NESTING_EVENT_TYPE		ENUM('TRANSACTION','STATEMENT','STAGE'));"
 
 // Fields in `events_statements_summary_by_digest` and `events_statements_summary_by_digest_history` are the same.
-const fieldsInEventsStatementsSummary = "SUMMARY_BEGIN_TIME TIMESTAMP(6) NOT NULL," +
-	"SUMMARY_END_TIME TIMESTAMP(6) NOT NULL," +
-	"STMT_TYPE VARCHAR(64) NOT NULL," +
-	"SCHEMA_NAME VARCHAR(64) DEFAULT NULL," +
+const fieldsInEventsStatementsSummary = "SUMMARY_BEGIN_TIME TIMESTAMP(6) NOT NULL COMMENT 'Begin time of this summary'," +
```
Comment (AilinKid) [inline infoschema/perfschema/const.go:388]: should this comments be the same as MySQL's?

Comment (djshow832) [reply infoschema/perfschema/const.go:388]: There's no comment in MySQL. And we have much more fields than that in MySQL, which must be explained.

Comment (AilinKid) [reply infoschema/perfschema/const.go:388]: sounds ok

**Pull request merged**